vignettes/adsl.Rmd

```diff
@@ -677,41 +677,37 @@ dataset_vignette(
 
 Numeric and categorical variables (`AGE`, `RACE`, `COUNTRY`, etc.) may need to be grouped to perform
 the required analysis.
-`{admiral}` does not **currently** have functionality to assist with all required groupings. So, the user will often need to create his/her own function to meet his/her study requirement. 
```

Comment:
Not sure if using this function in this circumstance is an improvement, as the result depend on the order. 

@bundfussr wdyt?

Also, it would be cool to do both AGEGR and REGION in just one lookup table, e.g. if we set the not-respective categorisation to NULL or something.. 
Although if you have really many variables to derive, that would expand the lookup table a lot, so in that case it would not make sense... hmm...

Reply:
I think the new function is useful if you need both the character and the numeric variable of a grouping. If you just need one of them, you could just use `mutate()` in combination with `if_else()`, `case_match()` or `case_when()`.

We could provide examples for both cases here.

I'm not sure if a big lookup table for many grouping variables would be useful. This seems to contradict the modularity approach of admiral.

Reply:
You are right. 
I added the previous version as an alternative below.